### ISSUE TITLE
Split shared/explorer test suite for parallelization

### ISSUE THREAD

**Issue by genehynson** [influxdata/ui#6278]
I noticed that shared/explorer tests are taking 10min while most other test suites are taking <5min. We should split the shared/explorer test suite into at least two files so those tests can be run in parallel. This would reduce the overall pipeline execution time by 5min. (30min --> 25min)

- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/17605/workflows/a10078bd-8bb1-49bc-9217-e144813b8a63
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/17605/workflows/a10078bd-8bb1-49bc-9217-e144813b8a63/jobs/1241539

**Comment by wiedld** [comment]
Already done. Closing.
